src/widget.rs

```diff
@@ -16,13 +17,30 @@ use pango;
 use ffi;
 
 use {
+    Continue,
     DestDefaults,
     Inhibit,
     Rectangle,
     TargetEntry,
     Widget,
 };
 
+/// The id of tick callback returned by `add_tick_callback`.
```

Comment:
@sfanxiang Sorry, we store comment in other crate https://github.com/gtk-rs/lgpl-docs,
so please, remove it after https://github.com/gtk-rs/lgpl-docs/pull/52 merged, it fix CI doctest

Reply:
Done